cloud/cloud.go

```diff
@@ -355,6 +403,12 @@ func Put[T Resource](ctx context.Context, client *Client, payload interface{}, u
 	return resource, nil
 }
 
+// Delete requests the endpoint url with a DELETE method.
+func Delete[T Resource](ctx context.Context, client *Client, url url.URL) error {
+	_, err := Request[T](ctx, client, "DELETE", url, nil)
```

Comment:
_nitpick_ Why don't you use `http.Method*` constants?
```suggestion
	_, err := Request[T](ctx, client, http.MethodDelete, url, nil)
```

Reply:
I actually don't know :) but I used the plain string in all other methods so I will keep it this way for now for consistency.